web-ui/src/app/components/latest-blocks/latest-blocks.component.html

```diff
@@ -26,7 +26,7 @@ <h4>{{'message.loadingLatestBlocks' | translate}}</h4>
           </thead>
 
           <tbody>
-            <tr *ngFor="let item of blocks">
+            <tr *ngFor="let item of blocks; let i = index" [ngClass]="{ 'latest-block': isBlockRecent(i) }">
```

Comment:
The index is not required, you can just call `isNewBlock(item)` where that function returns `true` if the item hash or height exists in the `newBlocks` variable, this can be done easily by the filtering over it.